Source/Routing and Faulting/Routing and Faulting Custom Device.lvproj

```diff
@@ -37,9 +48,7 @@
 					<Item Name="Routing and Faulting Scripting System Tests.lvclass" Type="LVClass" URL="../Scripting/Tests/System/Routing and Faulting Scripting System Tests.lvclass"/>
 				</Item>
 				<Item Name="Unit" Type="Folder">
-					<Item Name="Assets" Type="Folder">
-						<Item Name="Base System Definition.nivssdf" Type="Document" URL="../Scripting/Tests/Unit/Assets/Base System Definition.nivssdf"/>
```

Comment:
It looks like this asset still exists, it is just located in a different location:
`../Scripting/Tests/Unit/Routing and Faulting Scripting Unit Tests/Assets/Base System Definition.nivssdf`.